test/e2e_node/critical_pod_test.go

```diff
@@ -78,15 +76,24 @@ var _ = framework.KubeDescribe("CriticalPod [Serial] [Disruptive] [NodeFeature:C
 					v1.ResourceMemory: resource.MustParse("100Mi"),
 				},
 			})
+
 			nonCriticalBestEffort := getTestPod(false, bestEffortPodName, v1.ResourceRequirements{})
+			capacity := getNodeCPUAndMemoryCapacity(f)
+			used := v1.ResourceList{
+				v1.ResourceCPU:    resource.MustParse("200m"),
+				v1.ResourceMemory: resource.MustParse("200Mi"),
```

Comment:
where did these numbers come from?

Reply:
Looks like they are random. They should however not be used, since the critical pod _should_ request all the available resources on the node in order to force preemption of other pods.

Reply:
This smells like particular values were chosen to work with a particular test environment, and would be inaccurate or lead to test failures in differently sized CI environments. Is that the case?

Reply:
It shouldn't request all _available_ resources.  It should request _all_ (allocatable) resources on the node.  We shouldn't subtract anything.